### PR TITLE
mgmt, search, add live tests for issue

### PR DESCRIPTION
**Pull request by v-hongli1**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/resourcemanager/azure-resourcemanager-search/src/test/java/SearchTests.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        final AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
+
+        searchServiceManager = SearchServiceManager
+            .configure().withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile);
+
+        resourceManager = ResourceManager
+            .configure().withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile)
+            .withSubscription("faa080af-c1d8-40ad-9cce-e1a450ca5b57");
```
Comment (weidongxu-microsoft) [inline sdk/resourcemanager/azure-resourcemanager-search/src/test/java/SearchTests.java:43]: Just with `withDefaultSubscription()`, which uses the env variable (a hardcoded id may not work if dev run it locally on their different subscription).

**Comment by XiaofeiCao** [comment]
@weidongxu-microsoft Search is premium SDK. Thought we only need live tests for lite?

File: sdk/resourcemanager/azure-resourcemanager-search/src/test/java/SearchTests.java (hunk 1 above)
Comment (v-hongli1) [reply sdk/resourcemanager/azure-resourcemanager-search/src/test/java/SearchTests.java]: Fixed in the new version.

**Comment by weidongxu-microsoft** [comment]
Yes, sorry. @v-hongli1 Please close this one. Premium SDK already had recorded tests.

I will delete the CI pipelines.

**Comment by v-hongli1** [comment]
The `azure-resourcemanager-search` is a premium SDK and already had recorded tests, so cancel fix for live test.

**Pull request closed without merging**